episimlab/partition/travel_management.py

```diff
@@ -0,0 +1,56 @@
+import pandas as pd
```

Comment:
@kellypierce Is this intentional merge, or perhaps leftover from #18? Do you think we need the `TravelManager`?

Reply:
It's a leftover! Thanks for catching. I'll remove TravelManager.